core/blockchain.go

```diff
@@ -1465,12 +1467,18 @@ func (bc *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 	}
 	// Ensure the user sees large reorgs
 	if len(oldChain) > 0 && len(newChain) > 0 {
-		logFn := log.Debug
+		logFn := log.Info
```

Comment:
hm, I wonder if this isn't a bit too verbose

Reply:
for ethereum-parity cases, all reorg logged like as following
![image](https://user-images.githubusercontent.com/32324335/51842249-ca9c2800-2353-11e9-8b1f-4dc4da33d25c.png)